### PR TITLE
Update QUnit

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

**Comment by mightyiam** [comment]
@winhamwr the module selection drop down thing was broken. Selecting a module in master results in zero tests run.

I took this opportunity to check whether updating QUnit solves this. It does.

So I didn't investigate further.

**Comment by mightyiam** [comment]
Apparently, it doesn't seem to solve the issue. Perhaps something is inconsistent.

**Comment by mightyiam** [comment]
The issue seems unrelated to QUnit. A trailing slash is added in some browsers.

File: src/test/unit/manipulation-test-helper.js [hunk 1]
```diff
@@ -2,9 +2,10 @@
     manipulationTestHelper
 */
 /* global
-    expect,
     wymEqual,
     QUnit,
+    expectOneMore,
```
Comment (winhamwr) [inline src/test/unit/manipulation-test-helper.js]: Nice helper.

File: bower.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "jquery": "~1.4.4",
     "jquery.browser": "~0.0.6",
     "js-beautify": "https://github.com/beautify-web/js-beautify.git#1.5",
+    "qunit": "~1.17.1",
```
Comment (winhamwr) [inline bower.json]: Bower all the things!

**Pull request merged**